api/api.yaml

```diff
@@ -2641,6 +2642,19 @@ components:
           type: string
           description: The time the project was updated.
           format: date-time
+        entitlements:
+          type: object
+          description: The entitlements for the project.
+          properties:
+            brandingRemoval:
+              type: object
+              x-go-type-skip-optional-pointer: true
+              description: Whether branding has been removed from the project.
+              properties:
+                value:
```

Comment:
value should probably be `required`, then you don't need the ` x-go-type-skip-optional-pointer: true`